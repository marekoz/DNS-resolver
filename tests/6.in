2001:4860:4860:0000:0000:0000:0000:8888 -x  -r -s kazi.fit.vutbr.cz