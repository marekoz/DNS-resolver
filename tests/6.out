Authoritative: No, Recursive: Yes, Truncated: No
Question section (1)
  8.8.8.8.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.6.8.4.0.6.8.4.1.0.0.2.ip6.arpa., Type: PTR, Class: IN
Answer section (1)
  8.8.8.8.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.6.8.4.0.6.8.4.1.0.0.2.ip6.arpa., Type: PTR, Class: IN, TTL: <number>, Data length: 12, dns.google.
Authority section (0)
Additional section (0)